src/Agent.Plugins/PipelineArtifact/FilePathUNCServer.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Agent.Sdk;
+using Microsoft.TeamFoundation.Build.WebApi;
+using Microsoft.VisualStudio.Services.WebApi;
+using Microsoft.VisualStudio.Services.Agent.Util;
+
+namespace Agent.Plugins.PipelineArtifact
+{
+    // A wrapper of DedupManifestArtifactClient, providing basic functionalities such as uploading and downloading pipeline artifacts.
```

Comment:
How is this a wrapper of DedupManifestArtifactClient ?

Reply:
pipeline artifacts -> "File share"

Reply:
Have updated the comments but not pushed yet.